src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/MauiCustomViewTarget.java

```diff
@@ -57,10 +58,10 @@ private void post(Runnable runnable) {
     }
 
     private void clear() {
-        post(() -> {
+        //post(() -> {
```

Comment:
Leaving the comments here for future reference; I don't think we actually need them, but that's beyond the scope of this change. Commenting out the empty post() for performance, though.